README.md

```diff
@@ -1036,7 +1055,7 @@ function getValue(MyObject $object)
 /**
  * @throws PaybackUnavailableException
  */
-function payback($requestId)
+function payback($requestId): void
```

Comment:
in some place we add that requestId is int, here not, maybe we should keep consistency